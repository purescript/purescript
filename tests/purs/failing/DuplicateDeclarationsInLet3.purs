-- @shouldFailWith OverlappingNamesInLet
-- @shouldFailWith OverlappingNamesInLet
module Main where

-- Should see separate errors for `a` and `interrupted`
foo = interrupter + a
  where
  a = 0
  a :: Int
  a = 0

  interrupted true = 1

  interrupter = 2

  interrupted false = 3
