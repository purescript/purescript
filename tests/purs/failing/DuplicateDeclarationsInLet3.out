Error 1 of 2:

  in module [33mMain[0m
  at tests/purs/failing/DuplicateDeclarationsInLet3.purs:9:3 - 9:11 (line 9, column 3 - line 9, column 11)

    The name [33ma[0m was defined multiple times in a binding group


  See https://github.com/purescript/documentation/blob/master/errors/OverlappingNamesInLet.md for more information,
  or to contribute content related to this error.

Error 2 of 2:

  in module [33mMain[0m
  at tests/purs/failing/DuplicateDeclarationsInLet3.purs:16:3 - 16:24 (line 16, column 3 - line 16, column 24)

    The name [33minterrupted[0m was defined multiple times in a binding group


  See https://github.com/purescript/documentation/blob/master/errors/OverlappingNamesInLet.md for more information,
  or to contribute content related to this error.

